src/GeositeFramework/css/main.css

```diff
@@ -2335,3 +2335,7 @@ body.x-body {
 .launchpad-plugin li p {
   font-size: 13px;
 }
+.icon-wrench:before {
```

Comment:
Sounds good. I pushed a fixup which adds the id and classname from the parent button to ensure we're only applying the rule here.